src/builder/commands/generic.rs

```diff
@@ -84,14 +83,17 @@ fn find_cmd<P:AsRef<Path>>(ctx: &Context, cmd: P)
     if let Some(paths) = ctx.environ.get("PATH") {
         for dir in paths[..].split(':') {
             let path = Path::new(dir);
-            if !path.is_absolute() {
-                warn!("All items in PATH must be absolute, not {}",
-                      path.display());
-                continue;
-            }
-            if chroot.join(path.rel()).join(cmd).exists()
-            {
-                return Ok(path.join(cmd));
+            match path.strip_prefix("/") {
+                Ok(rel_path) => {
+                    if chroot.join(rel_path).join(cmd).exists() {
+                        return Ok(path.join(cmd));
+                    }
+                },
+                Err(_) => {
+                    warn!("All items in PATH must be absolute, not {}",
+                        path.display());
```

Comment:
better use `{:?}` instead of path display. The `path.display()` is legacy from 1.0 rust. 
(the reason is if path something weird like `\n` debug display will show it escaped, but `path.display()` prints a broken line)